src/shims/x86/mod.rs

```diff
@@ -95,11 +95,22 @@ pub(super) trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 }
             }
 
-            "pclmulqdq" => {
+            "pclmulqdq" | "pclmulqdq.256" | "pclmulqdq.512" => {
+                let mut width = 2;
+                this.expect_target_feature_for_intrinsic(link_name, "pclmulqdq")?;
+                if unprefixed_name.ends_with(".256") {
+                    this.expect_target_feature_for_intrinsic(link_name, "vpclmulqdq")?;
```

Comment:
Is there a way to test this without requiring avx512f? The current test uses __m512 types even to test the 256bit version of the intrinsic.

Reply:
At minimum it is possible to split the test into two, one for testing with fvx512f, one without. My personal preference however would be to run the same test twice with different compiler flags. Is there a way to do this?

Reply:
Yes, that can be done with revisions. Grep the test folder for `@revisions` to see some examples.

Reply:
Done.